### ISSUE TITLE
Support for other YAML attributes

### ISSUE THREAD

**Issue by tonygrr** [oleeskild/obsidian-digital-garden#18]
Hi. Thank you for your work, this is the easiest, most convenient and easily customizable way to publish your notes that I have tried. 

Is it possible in the future to make support for other YAML attributes together with the attributes for publishing? Something like this:

```
---
dg-publish: true
date: 2022.03.12
time: 19:58
aliases: []
tags: []
---
```

Right now if I try to publish a note with such metadata, I get errors in the deployment log.

**Comment by oleeskild** [comment]
Hi! Thanks, happy to hear you like it.

I wasn't aware of this, thanks for letting me know. I've released a new version of the plugin (2.6.1) which should fix this. Let me know if it works for you.

**Comment by tonygrr** [comment]
@oleeskild Thank you so much! Everything works great.
